server.js

```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 const CollectionMicrositeService = require("./collectionMicrosite.service.js");
-require('dotenv').config({ path: __dirname + '/config/.env' });
+require('dotenv').config({path: __dirname + '/config/.env'});
```

Comment:
Could you configure your IDE based config from yonote? 
https://floteam.yonote.ru/doc/zametki-po-strukture-koda-wp5b2OrxCn